packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import fs from 'fs';
+import path from 'path';
+
+import { rollup, RollupLog } from 'rollup';
+// @ts-ignore
```

Comment:
That'd be a good addition, even for my own sanity. Because I can't recall why I added that here. I'll either replace with `@ts-expect-error` with an actual reason, or remove it altogether.